src/Microsoft.AspNet.Mvc.Core/UrlHelper.cs

```diff
@@ -68,8 +68,13 @@ public UrlHelper(IScopedInstance<ActionContext> contextAccessor, IActionSelector
             return GenerateUrl(protocol, host, path, fragment);
         }
 
-        /// <inheritdoc />
-        public bool IsLocalUrl(string url)
+        /// <summary>
+        /// Returns a value that indicates whether the URL is local.
+        /// </summary>
+        /// <param name="url">The URL.</param>
+        /// <returns>true if the URL is local; otherwise, false.</returns>
+
+        public static bool IsLocalUrl(string url)
```

Comment:
@yishaigalatzer we were discussing making this method back into a static. It's a security feature and having it show up as mockable seemed bad.